src/lib/tableland-calls.ts

```diff
@@ -31,38 +36,49 @@ async function sendResponse(res: any) {
   return json.result;
```

Comment:
Could we put the 'camelCaseKeys' function here. and remove it from the other functions? It looks like anywhere "SendCall" is used, the response is wrapped in `camelCaseKeys` anyway.

Reply:
Good call!